### PR TITLE
Feature/error

### PR DESCRIPTION
**Pull request by tim-nguyen-cs**
Problem
=======
As a prospective hacker,
I would like useful error webpage
To understand when I've incorrectly navigated a website.

[Link to Asana Ticket](https://app.asana.com/0/1197738625416321/1199387105632109/f)

Solution
========
To implement the Error section of the webpage, flexbox-based and relative/absolute positioning and component styling were used to replicate the provided Figma design with responsiveness. Specifically, a secondary image layer was integrate beneath the preliminary content layer to showcase appropriate lens flares and to position them correctly. In the content layer, error title, image, and button were laid out using margins, but may be revised to incorporate more flexbox-based positioning types, such as `space-evenly` or `space-between`.

Change summary:
---------------
* boiler-plated 404 page c4e516e
* added images and improved responsiveness 0c13baf
* fixed image layering 44f8418

Steps to Verify:
----------------
1. Checkout branch using `git checkout feature/error`
2. Start local development server using `yarn start`
3. Navigate to any URL not corresponding to the homepage (`/`) to verify that error section is designed and responds correctly.
4. Click the "let's go home" button to verify that all error pages are correctly redirected to the homepage.

### CODE REVIEW

File: src/views/Error/Error.scss [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    left: 0;
+    height: 100%;
+    width: 100%;
+
+    background: linear-gradient(180deg, var(--aubergine) 0%, var(--haiti) 100%);
+    overflow: hidden;
+    z-index: -1;
+  }
+
+  &__lensFlare--left {
```
Comment (mocklee) [inline src/views/Error/Error.scss:72]: these css props for lens flares can be consolidated using a mixin by passing in the values as arguments

**Review by mocklee — APPROVED** [review]
looks perfect in the browser! let's roll this into staging

File: src/stylesheets/_fonts.scss [hunk 2]
```diff
@@ -39,3 +39,10 @@
     url("~fonts/montserrat-v15-latin-500italic.ttf") format("truetype"),
     url("~fonts/montserrat-v15-latin-500italic.svg#Montserrat") format("svg"); /* Legacy iOS */
 }
+
+/* source-code-pro-regular */
+@font-face {
+  font-family: "Source Code Pro";
+  font-weight: 400;
+  src: url("https://fonts.googleapis.com/css2?family=Source+Code+Pro&display=swap");
+}
```
Comment (ilopezro) [inline src/stylesheets/_fonts.scss:48]: Let's wait to push this in until we have Priya's PR in. She has the entire fonts stores locally so we can get rid of this once that PR is in.

File: src/views/Error/Error.scss (hunk 1 above)
Comment (tim-nguyen-cs) [reply src/views/Error/Error.scss:81]: I added a lens flare mixin in SCSS! Let me know how it looks!

**Pull request merged**